src/AWS.Deploy.Constants/ElasticBeanstalk.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AWS.Deploy.Constants
+{
+    internal static class ElasticBeanstalk
+    {
+        public const string EnhancedHealthReportingOptionId = "EnhancedHealthReporting";
+        public const string EnhancedHealthReportingOptionNameSpace = "aws:elasticbeanstalk:healthreporting:system";
+        public const string EnhancedHealthReportingOptionName = "SystemType";
+
+        public const string XRayTracingOptionId = "XRayTracingSupportEnabled";
+        public const string XRayTracingOptionNameSpace = "aws:elasticbeanstalk:xray";
+        public const string XRayTracingOptionName = "XRayEnabled";
+
+        public const string ProxyOptionId = "ReverseProxy";
+        public const string ProxyOptionNameSpace = "aws:elasticbeanstalk:environment:proxy";
+        public const string ProxyOptionName = "ProxyServer";
+
+        public const string HealthCheckURLOptionId = "HealthCheckURL";
+        public const string HealthCheckURLOptionNameSpace = "aws:elasticbeanstalk:application";
+        public const string HealthCheckURLOptionName = "Application Healthcheck URL";
+
+        public static List<Tuple<string, string, string>> OptionSettingQueryList = new()
+        {
+            new (EnhancedHealthReportingOptionId, EnhancedHealthReportingOptionNameSpace, EnhancedHealthReportingOptionName),
+            new (XRayTracingOptionId, XRayTracingOptionNameSpace, XRayTracingOptionName),
+            new (ProxyOptionId, ProxyOptionNameSpace, ProxyOptionName),
+            new (HealthCheckURLOptionId, HealthCheckURLOptionNameSpace, HealthCheckURLOptionName)
+        };
```

Comment:
I'd prefer if we don't use a Tuple. They are very vague to work with. Either replace this by a class or add some documentation to explain what the different values are.